interface/weno/adminClass.php

```diff
@@ -0,0 +1,740 @@
+                        `Mail_Address_Line_2` varchar(16) DEFAULT NULL,
+                        `Mail_City` varchar(10) DEFAULT NULL,
+                        `Mail_State` varchar(2) DEFAULT NULL,
+                        `Mail_Zip_Code` int(9) DEFAULT NULL,
+                        `Mail_Phone` int(10) DEFAULT NULL,
+                        `Mail_Fax` int(10) DEFAULT NULL,
+                        `EPCS_Permitted` varchar(5) DEFAULT NULL,
+                        `Accept_NewRx` varchar(4) DEFAULT NULL,
+                        `Accept_Refillresponse` varchar(5) DEFAULT NULL,
+                        `Accept_RxChangeresponse` varchar(5) DEFAULT NULL,
+                        `Accept_Verify` varchar(5) DEFAULT NULL,
+                        `Accept_Cancelrx` varchar(5) DEFAULT NULL,
+                        `Accept_RxHistoryrequest` varchar(5) DEFAULT NULL,
+                        `Accept_RxHistoryresponse` varchar(5) DEFAULT NULL,
+                        `Accept_Census` varchar(5) DEFAULT NULL,
+                        `Accept_Resupply` varchar(5) DEFAULT NULL,
+                        PRIMARY KEY (`id`)
+                        ) ENGINE=InnoDB  DEFAULT CHARSET=utf8 AUTO_INCREMENT=11150"
+    		);
+
+          sqlStatement(
+            "CREATE TABLE IF NOT EXISTS `erx_rx_log` (
+                         `id` int(20) NOT NULL AUTO_INCREMENT,
+                         `prescription_id` int(6) NOT NULL,
+                         `date` varchar(25) NOT NULL,
+                         `time` varchar(15) NOT NULL,
+                         `code` int(6) NOT NULL,
+                         `status` text NOT NULL,
+                         `message_id` varchar(100) DEFAULT NULL,
+                         `read` int(1) DEFAULT NULL,
+                         PRIMARY KEY (`id`)
+                          ) ENGINE=InnoDB  DEFAULT CHARSET=utf8 AUTO_INCREMENT=133" 
+          	);
+
+          sqlStatement(
+              "CREATE TABLE IF NOT EXISTS `erx_erx_narcotics` (
+                            `id` int(11) NOT NULL AUTO_INCREMENT,
+                            `drug` varchar(255) NOT NULL,
+                            `dea_number` varchar(5) NOT NULL,
+                            `csa_sch` varchar(2) NOT NULL,
+                            `narc` varchar(2) NOT NULL,
+                            `other_names` varchar(255) NOT NULL,
+                             PRIMARY KEY (`id`)
+                           ) ENGINE=InnoDB  DEFAULT CHARSET=utf8 AUTO_INCREMENT=570"
+          	);
+
+          sqlStatement("ALTER TABLE `users` ADD `weno_prov_id` VARCHAR(15) NULL AFTER `newcrop_user_role`");
+          
+          sqlStatement("ALTER TABLE `prescriptions` ADD `ntx` INT(2) NULL DEFAULT NULL AFTER `prn`, ADD `rtx` INT(2) NULL DEFAULT NULL AFTER `ntx`, ADD `txDate` DATE NOT NULL AFTER `rtx`");
+        
+          sqlStatement("ALTER TABLE `pharmacies` ADD `ncpdp` INT(12) NULL AFTER `email`, ADD `npi` INT(12) NULL AFTER `ncpdp`");
```

Comment:
First need to use standard practices for new 5.0.1 users and users upgrading to 5.0.1. Thus need to place these tables and tables mods in database.sql and the standard upgrade script for 5.0.0 to 5.0.1. 
I think what you are asking here is what to do for 5.0.0 users that want to use this feature. And in that case you can do something like what you are doing here (but that code would not go into the main codebase since it's not pertinent).